Week02/README.md

```diff
@@ -23,6 +23,13 @@
 - [Complicated GCD](http://codeforces.com/contest/664/problem/A)
     - [Bedir's Solution](https://github.com/BedirT/AlgorithmsL/blob/master/Problems/Curriculum%20Q's/Week%201/Complicated%20GCD.cpp)   
 
+### Advanced GCD
+
+#### [Tutorial](https://www.geeksforgeeks.org/gcd-of-two-numbers-when-one-of-them-can-be-very-large-2/)
```

Comment:
Tutorials are the specific blog posts prepared only for this guide. Geeks for Geeks goes under **Study Sources**

Reply:
@BedirT sir, ok i will take care about it, next time